gateway/src/node/mod.rs

```diff
@@ -623,6 +624,11 @@ impl<St> Gateway<St> {
                 return Err(GatewayError::InsufficientNodeBalance { account, balance });
             }
         }
+        let shared_session_stats = self.session_stats.take().unwrap_or_default();
+        let stats_event_sender = self.start_stats_collector(
+            shared_session_stats,
+            shutdown.fork("statistics::sessionCollector"),
```

Comment:
See comment in the other review about the name :)

Reply:
Fixed by the rebasing of this PR on top of the other one